File: Java/Java source-code/distanceInf.java


public class distanceInf {
    public static double distanceInf(Double[] p1, Double[] p2) {
        double max = 0;
        for (int i = 0; i < p1.length; i++) {
            max = Math.max(max, Math.abs(p1[i] - p2[i]));
        }
        return max;
    }
}

File: Methods-C++/distanceInf.cpp
#include <iostream>
#include <math.h>
#include <algorithm>

double distanceInf(double p1[], double p2[], int n) {
    double maxi = 0;
    for (int i = 0; i < n; i++) {
        maxi = max(maxi, abs(p1[i] - p2[i]));
    } 
    return maxi;
}